styles.css

```diff
@@ -29,14 +19,27 @@ body {
   justify-content: space-evenly;
   width: 100vw;
   height: 100vh;
-  background: linear-gradient(var(--primary), var(--secondary), var(--third));
+  /* background: linear-gradient(var(--primary), var(--secondary), var(--third)); */
```

Comment:
remove this commented out line

Reply:
Removed the commented line 22, pushed the changes for review!